server/src/yaml-utils.ts

```diff
@@ -20,6 +20,7 @@ export class YAMLSourceMap {
     this.nodes = new Map<string, TokenRange>();
   }
 
+  /* eslint-disable  @typescript-eslint/no-explicit-any */
   public doMap(node: any | null, path: string[] = []) {
```

Comment:
Was not sure about the `node` type here, that's why needed to add this explicit warning suppresser.